server/src/test/java/io/crate/integrationtests/SysSnapshotsTest.java

```diff
@@ -62,7 +64,7 @@ protected Settings nodeSettings(int nodeOrdinal) {
     @Test
     @SuppressWarnings("unchecked")
     public void testQueryAllColumns() throws Exception {
-        execute("create table tbl (id int primary key) ");
+        execute("create table tbl (id int primary key) clustered by (id) into 4 shards");
```

Comment:
I guess you want to explicitly ensure that there are 4 shards. This is the default, but setting it explicitly as it is asserted may be better. But why the `clustered by(id)` change? Nothing in this test will check for the routing column afaik. Also routing by the primary key is the default.
```suggestion
        execute("create table tbl (id int primary key) clustered into 4 shards;
```

Reply:
>This is the default

it was sometimes different form 4 because of random number of data nodes

Reply:
jfyi: number of data nodes aren't related to number of shards ;)